Makefile

```diff
@@ -61,7 +63,15 @@ clean:
 	$(GOCLEAN)
 	rm -f $(PROGRAM)
 
+# Download the installation script for golangci-lint, verify its SHA-256 digest,
+# run it if everything checks out, and verify the resulting binary.
 $(GOLANGCI_LINT):
-	curl -sfL \
-			https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh \
-		| sh -s -- -b $(GOBIN) $(GOLANGCI_LINT_VERSION)
+	mkdir -p $(dir $@)
+	curl -sSfL \
+		https://raw.githubusercontent.com/golangci/golangci-lint/$(GOLANGCI_LINT_VERSION)/install.sh \
+		> $@.install-script-unverified
+	echo "$(GOLANGCI_LINT_INSTALLATION_SHA256) $@.install-script-unverified" | sha256sum -c --quiet -
+	sh -s -- -b $(dir $@) $(GOLANGCI_LINT_VERSION) < $@.install-script-unverified
+	rm -f $@.install-script-unverified
```

Comment:
Should you append this line onto 73 like you do on 76?

Reply:
I pushed a commit for that just now but realized that we'd have to add it to line 74 as well if we don't want to risk leaving an unverified but non-executable file with "unverified" in the name. It gets pretty noisy and I'm not sure I think it's worth it.

This case is different from line 76, where a failure would leave us with an executable `$@` in place, i.e. a subsequent `make` invocation wouldn't cause this rule to run again and the bad binary would be run. That's dangerous, but for the installation script you have to work pretty hard to run an unverified one.

Reply:
Yeah, got a notification that Magnus pushed 0 commits! 
You know you are productive when you get an email saying that nothing has been done :P
But alright, I support your reasoning :)